packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -226,6 +226,15 @@ class DescopeWc extends BaseDescopeWc {
     const loginId = this.sdk.getLastUserLoginId();
     const flowConfig = await this.getFlowConfig();
     const projectConfig = await this.getProjectConfig();
+    const flowVersions = {
+      version: flowConfig.version,
+      componentsVersion: projectConfig.componentsVersion,
+      versions: Object.keys(projectConfig.flows || []).reduce((acc, key) => {
+        const flowConf = projectConfig.flows[key];
```

Comment:
not critical at all, but I lov to use `ObjectEntries([key, value] => {}` that already get the matching value 
versions: Object.values(projectConfig.flows || []).reduce((acc, [key, flowConf) => {
...}, {})